tests/conftest.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+sys.path.append(os.path.join(os.path.dirname(__file__), 'utils'))
```

Comment:
We are not pytest.ini for backwards compatibility and what not, is that correct?

Reply:
ironically these are all pytest fixtures 🤔